.gitignore

```diff
@@ -19,3 +19,7 @@ tmp/
 # Thyra builded react files
 int/api/dist/
 main
+
+# Go swagger generated files
+api/swagger/server/models/
+api/swagger/server/restapi/operations/
```

Comment:
I guess this can deal with all sub dirs:

```suggestion
**/api/server/restapi
```

Reply:
The swagger configuration is in .../restpi/resource/swagger.yml
We can move it, but we can't git-ignored it.

Reply:
Ah, I see! I would move it and ignore the whole dire then...

Anyway, non blocking...

Reply:
Feel free to resolve, of course!

Reply:
I do not really feel the need of generic path search. as far as i know we have only 1 go swagger api and we know its files path